recipes/cofold/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# ViennaRNA does not support Python 3 yet.
+./configure --prefix=$PREFIX --without-perl -q
+make
```

Comment:
The package contains some precompiled libraries (libRNA.a) compiled with gcc<2.7.  You can easiliy get rid of it by running:

```
./configure etc etc
make clean
make
make install
```

Maybe this solves your travis issue?